core/tests/protractor_utils/CreatorDashboardPage.js

```diff
@@ -102,6 +102,8 @@ var CreatorDashboardPage = function() {
   };
 
   this.navigateToCollectionEditor = async function() {
+    await waitFor.visibilityOf(
+      collectionCard, 'Unable to find collection card');
```

Comment:
There's already a waitFor in the `action.click` function, so I don't think we need this

Reply:
removed!